future/src/ct/ct_main_win.cc

```diff
@@ -124,10 +124,17 @@ CtMainWin::CtMainWin(bool             start_hidden,
     menu_set_items_special_chars();
     _uCtMenu->find_action("ct_vacuum")->signal_set_visible.emit(false);
 
-    if (start_hidden || (_pCtConfig->systrayOn && _pCtConfig->startOnSystray))
-        set_visible(false);
-    else
+    if (start_hidden || (_pCtConfig->systrayOn && _pCtConfig->startOnSystray)) {
```

Comment:
start_hidden is used to hide windows during import from command line. It's not related to tray, so it has to work